doc/rst/tools/mason/mason.rst

```diff
@@ -41,6 +41,19 @@ To initialize a new mason package, run the ``mason new [ package name ] [ option
 
 This creates a git repository by default, unless ``--no-vcs`` is included.
 
+Mason packages can also be initialized using the ``mason init [options] [directory path]`` or 
+``mason init [options]`` command outside or inside the project directory respectively. 
+
+For example, for an existing directory named MyPackage, 
+    
```

Comment:
I think we will need a code-block for this to render, e.g.:

```
.. code-block: sh
    
    mason init MyPackage 

    # OR 

    cd MyPackage
    mason init

Reply:
done 👍